Source/swiftgen.yml

```
@@ -0,0 +1,8 @@
+xcassets:
+  inputs:
+    - OmegaBank/Resources/Assets.xcassets
+  outputs:
+    - templatePath: ./swift5.stencil
```

Попробуй сделать так, тогда *.stencil-файл не понадобится.
```suggestion
    - templateName: swift5
```